fw/http.c

```diff
@@ -5627,6 +5627,18 @@ __extract_request_authority(TfwHttpReq *req)
 static bool
 __check_authority_correctness(TfwHttpReq *req)
 {
+	int conn_type = TFW_CONN_TYPE(req->conn)
+			& (TFW_FSM_HTTPS | TFW_FSM_WEBSOCKET);
+	if (conn_type == TFW_FSM_HTTPS) {
```

Comment:
```suggestion
	if (TFW_CONN_TYPE(req->conn) & TFW_FSM_HTTPS) {
```
Isn't this the same?

Reply:
Initial code requires websocket bit to be off. From what I know websocket connection starts from a regular HTTP request.

Reply:
Yep, but we never call HTTP after an HTTP connection is upgraded to websocket in `tfw_http_resp_process()`, i.e. this code is never called fro websocket connections. You can use `BUG_ON()` to make sure that there is no websocket flag